dataland-frontend/src/components/resources/landing/LandingLogin.vue

```diff
@@ -47,31 +57,24 @@
           </Card>
         </div>
       </div>
```

Comment:
The section above contains either sections that are displayed if it's a mobile page, or if it's a desktop page. There's no overlap. 
What do you think about having two components: `LandingLoginTextMobile` and `LandingLoginTextDefault` or the, and display the one or the other component depending on isMobile?

Reply:
done

Reply:
thank you!